docs/install.md

```diff
@@ -99,3 +102,6 @@ export PYTHONPATH=$PYTHONPATH:path-to-install-folder/python
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:path-to-install-folder/lib
 ```
+If using within an Anaconda environment:
+export GDAL_DIR=base-anaconda-directory/envs/fringe/
```

Comment:
I was thinking of also making a small PR on this issue that I ran into- I ended up changing it at during the `cmake` command by adding 

`GDAL_DIR=`gdal-config --prefix`  cmake -DCMAKE_INSTALL_PREFIX=../install ..`

Reply:
Interesting, was this due to using a system installation of gdal? In my experience I always use gdal from conda so -DCMAKE_PREFIX_PATH=$CONDA_PREFIX is sufficient, but this is a neat trick in the general case.

Reply:
Oh you are right! I had mixed it up with another setup that didn't have the anaconda gdal. The `-DCMAKE_PREFIX_PATH=$CONDA_PREFIX` was indeed sufficient for the conda gdal install.